orca-front50/src/test/groovy/com/netflix/spinnaker/orca/front50/model/ApplicationNotificationsSpec.groovy

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.front50.model
+
+import com.netflix.spinnaker.orca.front50.model.ApplicationNotifications.Notification
+import spock.lang.Specification
+
+class ApplicationNotificationsSpec extends Specification {
+
+  void "getPipelineNotifications extracts only pipeline notifications"() {
```

Comment:
would like to see a test to indicate that only the notifications for that application are being added